app/assets/javascripts/discourse/components/user-card-contents.js

```diff
@@ -38,7 +38,11 @@ export default Component.extend(CardContentsBase, CanCheckEmails, CleansUp, {
   showMoreBadges: gt("moreBadgesCount", 0),
   showDelete: and("viewingAdmin", "showName", "user.canBeDeleted"),
   linkWebsite: not("user.isBasic"),
-  hasLocationOrWebsite: or("user.location", "user.website_name"),
+  hasLocaleOrWebsite: or(
+    "user.location",
+    "user.website_name",
+    "user._timezone"
```

Comment:
We should not be making computed properties dependant on private member variables.

We have to either convert `_timezone` to be a public variable or remove it from this.